src/platform-implementation-js/dom-driver/gmail/widgets/gmail-mole-view-driver.ts

```diff
@@ -162,7 +220,7 @@ class GmailMoleViewDriver {
       const moleParent = this.#element.parentElement;
 
       if (moleParent && this.#element.getBoundingClientRect().left < 0) {
-        moleParent.insertBefore(this.#element, last(moleParent.children)!);
+        moleParent.insertBefore(this.#element, moleParent.lastElementChild!);
```

Comment:
Do we want still want SDK moles to become the rightmost mole if they're programmatically unminimized? This code was added in 5678a73296c2914a994bfd472280febf81e3c1b3, 8 years ago.

This existing version of this code won't work as expected when Google Chat is enabled.